app/src/main/java/com/codesoom/assignment/application/ToyService.java

```diff
@@ -0,0 +1,46 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.domain.Product;
+import com.codesoom.assignment.domain.ToyRepository;
+import com.codesoom.assignment.error.NotFoundException;
+import com.codesoom.assignment.interfaces.ProductRepository;
+import com.codesoom.assignment.interfaces.ProductService;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class ToyService implements ProductService {
+    private final ProductRepository productRepository;
+
+    public ToyService(ToyRepository productRepository){
+        this.productRepository = productRepository;
+    }
+
+    @Override
+    public List<Product> findProducts() {
+        List<Product> products = productRepository.findAll();
+        return products;
```

Comment:
좌변에 변수를 선언하지 않아도 `products`가 반환될 것이라는 것은 여기 메서드의 이름과 `findAll`이라는 곳에서 이미 알고 있으므로, 바로 반환해줘도 될 것 같아요
```suggestion
        return  productRepository.findAll();
```